common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml

```diff
@@ -330,8 +330,8 @@ eActionOp_CLI:
   ACTION_CLI_CLIENT_DISALLOW_REQUEST: 81
   ACTION_CLI_CLIENT_DISCONNECT_REQUEST: 82
   ACTION_CLI_CLIENT_BSS_STEER_REQUEST: 83
-  ACTION_CLI_CLIENT_LINK_MEASUREMENT_11K_REQUEST: 84
 
+  ACTION_CLI_CLIENT_CHANNEL_LOAD_11K_REQUEST: 85
```

Comment:
this commit message is about removal - not addition of new CMDUs

Reply:
fixed